### ISSUE TITLE
stripe_portal_configuration issue on 1.6.3 version

### ISSUE THREAD

**Issue by galkin** [lukasaron/terraform-provider-stripe#30]
After upgrade from 1.6.0 to 1.6.3 I had this error:

```
│ Error: business_profile: '': source data must be an array or slice, got struct
│
│   with stripe_portal_configuration.portal_configuration,
│   on 12_stripe.tf line 21, in resource "stripe_portal_configuration" "portal_configuration":
│   21: resource "stripe_portal_configuration" "portal_configuration" {
│
╵
╷
│ Error: features: '': source data must be an array or slice, got struct
│
│   with stripe_portal_configuration.portal_configuration,
│   on 12_stripe.tf line 21, in resource "stripe_portal_configuration" "portal_configuration":
│   21: resource "stripe_portal_configuration" "portal_configuration" {
│
╵
```

The resource config is:
```terraform
resource "stripe_portal_configuration" "portal_configuration" {
  business_profile {
    privacy_policy_url   = "https://${local.domain_landing}/privacy"
    terms_of_service_url = "https://${local.domain_landing}/terms"
  }
  default_return_url = "http://${local.domain_app}/"
  features {
    customer_update {
      enabled         = true
      allowed_updates = ["email", "address"]
    }
    invoice_history {
      enabled = false
    }
    payment_method_update {
      enabled = true
    }
  }
}
```

Versions:
Terraform v1.3.9
on darwin_arm64
+ provider registry.terraform.io/hashicorp/google v4.55.0
+ provider registry.terraform.io/hashicorp/google-beta v4.55.0
+ provider registry.terraform.io/hashicorp/random v3.4.3
+ provider registry.terraform.io/integrations/github v5.18.0
+ provider registry.terraform.io/lukasaron/stripe v1.6.0
+ provider registry.terraform.io/trois-six/sendgrid v0.2.1

**Comment by lukasaron** [comment]
I'm planning to completely rework the Portal Configuration. Please stay tuned!

**Comment by lukasaron** [comment]
Was there any version in the past that worked for you? I am trying to find a quick solution for now and deal with the issue in my free time.
I wasn't personally implementing this resource but I am taking full responsibility for fixing the issue. 

@galkin and @preceptoralberto is this something with high priority for you?

**Comment by galkin** [comment]
@lukasaron it works for me with 1.6.0

**Comment by lukasaron** [comment]
@galkin @preceptoralberto Could you please try the updated version v1.7.1?

**Comment by galkin** [comment]
@lukasaron tested. The bug is present (((

**Comment by lukasaron** [comment]
A fix for this has been implemented in version v1.7.2 however when performing the plan it tries to remove the default values. Would need to investigate how to not populate those which are not required.

**Comment by galkin** [comment]
confirmed. @lukasaron thank you so much!
